sormas-api/src/main/java/de/symeda/sormas/api/customizableenum/CustomizableEnumFacade.java

```diff
@@ -46,6 +46,19 @@
 	 */
 	<T extends CustomizableEnum> T getEnumValue(CustomizableEnumType type, String value);
 
+	/**
+	 * Retrieves the cached content of a specific enum type. The result is already internationalized based
+	 * on the user's language, or the server language as a fallback. If this specific value has not been requested yet,
+	 * the cache is extended with it on demand.
+	 *
+	 * @param type
+	 *            The type for which to retrieve the enum value
+	 * @param <T>
+	 *            The specific extension of {@link CustomizableEnum} for type safety
+	 * @return The enum instances related to the type in parameter, internationalized caption, and optional properties
+	 */
+	<T extends CustomizableEnum> List<T> getEnumValues(CustomizableEnumType type);
```

Comment:
I don't think this method is necessary; you should simply be able to call getEnumValues(CustomizableEnumType, Disease) with disease == null instead. This new logic actually makes the diseases column unusable, but there might be cases in the future in which people want to associate a specific risk with a specific disease for whichever reason.